modules/pool/withdraw/components/PoolWithdrawSingleAsset.tsx

```diff
@@ -38,7 +41,9 @@ export function PoolWithdrawSingleAsset({ onShowPreview, ...rest }: Props) {
 
     //TODO: precision
     const isValid =
-        singleAssetWithdraw.amount === '' || parseFloat(singleAssetWithdraw.amount) <= parseFloat(maxAmount);
+        parseFloat(singleAssetWithdraw.amount) > 0 &&
+        parseFloat(singleAssetWithdraw.amount) <= parseFloat(maxAmount) &&
```

Comment:
Would be nice if you also fix the precision errors here since you're in the code already